wdl-lint/src/rules/command_mixed_indentation.rs

```diff
@@ -72,7 +72,7 @@ fn mixed_indentation(command: Span, span: Span, kind: IndentationKind) -> Diagno
             "this command section uses both tabs and spaces in leading whitespace",
             command,
         )
-        .with_fix("use the same whitespace character for indentation")
+        .with_fix("use either tabs or spaces for indentation")
```

Comment:
```suggestion
        .with_fix("use either tabs or spaces exclusively for indentation")
```

The current error makes leaves room for interpretation that they can be mixed tabs and spaces. Though, I do agree that enumerating the two characters is better than the former.